src/tests/ftest/launch.py

```diff
@@ -1561,6 +1564,8 @@ def _add_auto_storage_yaml(self, storage_info, yaml_dir, tier_0_type, scm_size,
             scm_size (int): scm_size to use with ram storage tiers
             scm_mount (str): the base path for the storage tier 0 scm_mount.
             max_nvme_tiers (int): maximum number of NVMe tiers to generate
+            control_metadata (str, optional): directory to store control plane metadata when using
+                metadata on SSD. Defaults to None.
```

Comment:
The caller defaults this to `None`, but not this function. Should it?

Reply:
Probably not. I can fix the typo later.